packages/main/src/themes/ListItemBase.css

```diff
@@ -9,6 +9,33 @@
 	border-bottom: 1px solid transparent;
 }
 
+/* highlight */
+.ui5-li-highlight {
+	position: absolute;
+    width: 0.375rem;
+    bottom: 0;
+    left: 0;
+    top: 0;
+    border-inline-end: 0.0625rem solid var(--ui5-listitem-background-color);
```

Comment:
Then, these styles also don't belong here any more right? This is the ListItemBase.

Reply:
I missed that. Thanks.